src/battle_arcade_game_board.c

```diff
@@ -1228,11 +1228,11 @@ static void ov84_0223F2B4(GAME_BOARD_SUB_3E8 *work, Party *playerParty, Party *o
     ov84_0223F5E4(work, playerParty, opponentParty, type);
 
     for (i = 0; i < 11; i++) {
-        sub_0200ACF0(work->resourceObj[i][0]);
+        sub_0200ACF0(work->resourceObj[i][GF_GFX_RES_TYPE_CHAR]);
```

Comment:
out of scope but would it be worth changing the resource obj into a struct proper, it will prolly be more readable in the long run instead of having to use index consts each time but Im not sure what issues it might introduce